tests/gen_cert_test.py

```diff
@@ -181,3 +179,39 @@ def test_cert_date(self, force_date, locale, timezone, expected):
         """
         cert_date = get_cert_date(force_date, locale, timezone)
         assert_equal(expected, cert_date)
+
+
+@ddt
+class CertificateGenTests(unittest.TestCase):
```

Comment:
At this point, we may as well continue with the one-class-per-file layout and move this to its own file, `test_certificate_gen` or some such.